packages/ui/components/ProfileInfo.tsx

```diff
@@ -0,0 +1,39 @@
+import { Avatar, HStack, Text, VStack } from '@chakra-ui/react';
+
+type ProfileInfoProps = {
+  imageUrl?: string;
+  name: string;
+  email: string;
+  address?: string;
+  phoneNumber?: string;
+  children?: React.ReactNode;
+};
+
+export default function ProfileInfo({
+  imageUrl,
+  name,
+  email,
+  address,
+  phoneNumber,
+  children,
+}: ProfileInfoProps) {
+  return (
+    <HStack spacing={5} py={3.5} align="flex-start">
+      <Avatar src={imageUrl} boxSize="4.4rem" />
+      <VStack align="left" gap={0}>
+        <HStack align="center" spacing={3}>
+          <Text fontSize="md" fontWeight="semibold">
+            {name}
+          </Text>
+          {children}
+        </HStack>
+        <Text fontSize="sm" fontWeight="normal">
+          {email}
+        </Text>
+        <Text fontSize="sm" fontWeight="normal">
+          {address || phoneNumber}
+        </Text>
```

Comment:
`email`, `address`, `phoneNumber` 의 위치가 고정적인 부분이 살짝 아쉬운 것 같습니다!

공용컴포넌트인 만큼, 컴포넌트의 추후 확장성을 위해, 보여줄 데이터의 정보를 명시적으로 표현하지 않는 편이 더 좋을 것 같다고 생각합니다!

`infoTexts` 라는 이름의 text array 를 받아서 표시할 수 있다면, 데이터를 구체적으로 명시하지 않아도 되고, `infoText` 가 하나만 있는 경우(주소 정보가 없는 경우)에 대한 예외처리를 하지 않아도 되기 때문에, 조금더 확장성 있는 공용컴포넌트가 될 수 있을 것 같아서 제안해보았는데, 현지님께서는 어떻게 생각하시나요?

```suggestion
        {infoTexts.map((infoText) => (
          <Text fontSize="sm" fontWeight="normal">
            {email}
          </Text>
        ))}
```

Reply:
p4: 저도 컴포넌트가 도메인에 종속되지 않도록 혜진님이 제안한 방식대로 변경하면 어떨까 해요 :)

Reply:
컴포넌트에 대해서 잘 몰랐는데 데이터를 구체적으로 명시하지 않는게 좋은 방법이라는것을 알게되었습니다~
혜진님과 동환님의 조언되로 수정하겠습니다